src/screens/CreatePost/elements/CreatePollContainer/CreatePollContainer.android.js

```diff
@@ -68,7 +75,9 @@
       <TouchableOpacity style={S.polldurationbutton} onPress={() => setIsDurationModalShown(true)}>
         <View style={S.row}>
           <Text style={S.fillparenttext}>Duration</Text>
-          <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+          <View style={S.polldurationbuttonview(isAnonym)}>
+            <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+          </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isAnonym` prop is also used here for conditional rendering of styles. As mentioned before, ensure that all necessary props are passed when using this component.
```diff
-          <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+           <View style={S.polldurationbuttonview(isAnonym)}>
+             <Text style={S.polldurationbuttontext}>{getDurationTimeText(selectedtime)}</Text>
+           </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->